packages/exo/test/non-enumerable-methods.test.js

```diff
@@ -42,7 +42,7 @@ test('test defineExoClass', t => {
   t.throws(() => upCounter.incr(-3), {
     message: 'In "incr" method of (UpCounter): arg 0?: -3 - Must be >= 0',
   });
-  // @ts-expect-error bad arg
+  // FIXME typedef should catch bad arg
```

Comment:
Attn @erights, I think we can tolerate a regression in tsc catching a case here. Maybe we can downgrade FIXME to a note that we would not be surprised if ts-expect-error became necessary in a future tsc upgrade.